Getting a check failure during a guix build:
```
.
----------------------------------------------------------------------
Ran 1 test in 5.525s

OK
F
======================================================================
FAIL: test_ELF (__main__.TestSymbolChecks)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/distsrc-base/distsrc-dae34eb480c8-arm-linux-gnueabihf/./contrib/devtools/test-symbol-check.py", line 69, in test_ELF
    self.assertEqual(call_symbol_check(cxx, source, executable, []),
AssertionError: Tuples differ: (1, 'test3: export of symbol __cxa_begin_c[216 chars]OLS') != (0, '')

First differing element 0:
1
0

+ (0, '')
- (1,
-  'test3: export of symbol __cxa_begin_cleanup not allowed!\n'
-  'test3: export of symbol __cxa_call_unexpected not allowed!\n'
-  'test3: export of symbol __aeabi_atexit not allowed!\n'
-  'test3: export of symbol __cxa_type_match not allowed!\n'
-  'test3: failed EXPORTED_SYMBOLS')

----------------------------------------------------------------------
Ran 1 test in 1.004s

FAILED (failures=1)
make: *** [Makefile:1351: test-security-check] Error 1
```

--- a/contrib/devtools/test-security-check.py
+++ b/contrib/devtools/test-security-check.py
@@ -15,10 +15,10 @@ from utils import determine_wellknown_cmd
 def write_testcode(filename):
     with open(filename, 'w', encoding="utf8") as f:
         f.write('''
-    #include <stdio.h>
+    #include <cstdio>
     int main()
     {
-        printf("the quick brown fox jumps over the lazy god\\n");
+        std::printf("the quick brown fox jumps over the lazy god\\n");
         return 0;
     }
     ''')

--- a/contrib/devtools/test-symbol-check.py
+++ b/contrib/devtools/test-symbol-check.py
@@ -57,11 +57,11 @@ class TestSymbolChecks(unittest.TestCase):
         executable = 'test3'
         with open(source, 'w', encoding="utf8") as f:
             f.write('''
-                #include <stdio.h>
+                #include <cstdio>
 
                 int main()
                 {
-                    printf("42");
+                    std::printf("42");
                     return 0;
                 }
         ''')

